[task] complement closed server and client descriptions

jaime-ez opened the pull request.

yasserf commented:
AWESOME!

yasserf commented:
doesn't work when seleting via menu

timaschew commented:
Done, but in order to get it working via menu, we need to inject the javascript in our own context instead of the iframe context.

jaime-ez committed:
[task] complement closed server and client descriptions

yasserf commented:
awesome thank you!

probably worth mentioning the pause and resume part right? Since you can stop a connection without killing the client.

Also, since you are doing a pretty neat job on maintaining, should we remove the banner from the home page? Maybe deepstream can just live on as a decent niche tool. As long as dependencies are kept up to date on the core there doesn't seem to be much missing (and still miles ahead of other popular solutions). Doesn't mean I'll be actively contributing though (I'm almost live with my new project and it is time intense 🙈, as well as theres a few OS libraries I might spit out of there lol)

jaime-ez commented:
Ok I'll try the pause and resume (havent' used them) and then update the docs.

As you say the core works great and should live on for whoever finds a use for it. I'm thinking on spinning up a deepstream instance for a live example on the website, google compute engine has a free tier that could work for that, and thus hopefully encourage more people to try it out.

I'm ok with removing the banner, I can "maintain" as I have been doing it, although I must warn that between october-february I won't be able to contribute much. I manage a farm also :) and that's the busy period in the southern hemisphere.

On the website repo I'll keep on making pull requests before pushing direct changes when I think the change is relevant and could benefit from your input, if you don't answer, no problem, I'll take as an implicit ok and merge. For core repos I'll wait for your input. Is that ok with you?  I understand you won't be able to contribute more to deepstream, I just hope you'll be available to comment on more tricky issues if required. 

Glad to hear your new project is about to go live! Can you share more about it?

jaime-ez committed:
[fix] links

jaime-ez committed:
[task] pasue and resume

:100644 100644 f332e1c 5e8efa2 M	content/docs/40-client-js/00-client/index.md
:100644 100644 5201cf9 8fcf294 M	content/tutorials/00-concepts/20-connectivity/index.md
